samples/AspNetCore/ControllerSample/Controllers/SampleController.cs

```diff
@@ -80,5 +80,16 @@ public async Task<ActionResult<Account>> Withdraw(Transaction transaction, [From
             await state.SaveAsync();
             return state.Value;
         }
+
+        /// <summary>
+        /// Method for returning a BadRequest result which will cause Dapr sidecar to throw an RpcException
+        [HttpPost("throwException")]
+        public async Task<ActionResult<Account>> ThrowException(Transaction transaction, [FromServices] DaprClient daprClient)
+        {
+            Console.WriteLine("Enter ThrowException");
+            var task = Task.Delay(10);
+            await task;
+            return BadRequest(@"{""statusCode"": 400, ""message"": ""bad request""}");
```

Comment:
so rather than putting JSON text in a string, do this. 

`BadRequest(new { statusCode: 400, message: "bad request", });`

Reply:
fixed